### DISCUSSION TITLE
why does the mimic_icu only has 76,540 patients,but mimic_core has 523,740 patients.

### DISCUSSION THREAD

**Discussion by zhengxu931012** [MIT-LCP/mimic-code#1137] [MIMIC-IV] [answered]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Description of the issue, including:
* what you have tried
* references to similar issues
* queries demonstrating your question (if applicable)

**Comment by zhengxu931012** [comment]
why does the mimic_icu only has 76,540 patients,but mimic_core has 523,740 patients.

**Comment by SichengH** [comment, marked as answer]
Not all patients went to the ICU.
